cmd/troubleshoot/cli/run.go

```diff
@@ -55,15 +56,15 @@ func runTroubleshoot(v *viper.Viper, arg string) error {
 		httpClient = http.DefaultClient
 	}
 
-	collectorContent, err := loadSupportBundleSpec(v, arg)
+	collectorContent, err := supportbundle.LoadSupportBundleSpec(arg, httpClient)
```

Comment:
I find it odd that httpClient is a global variable, but is passed around as a one off argument for some functions.  How about making something similar to `k8sutil.GetRESTConfig()` but for http client?

Reply:
Yes, that's a great point.